lib/libraries/knex.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+const Knex           = require('knex');
```

Comment:
Sorry, I think I was a bit unclear when I told you about the aligning of equal signs. We like to align them within with block (where a "block" is separated by new lines). So this should look like:

```js
const Knex = require('knex');

const DatabaseConfig = require('../../db');
```